components/hab/src/cli.rs

```diff
@@ -351,21 +350,49 @@ pub fn get() -> App<'static, 'static> {
         )
         (subcommand: alias_apply)
         (subcommand: alias_install)
+        (subcommand: alias_run())
         (subcommand: alias_setup)
-        (subcommand: alias_start)
+        (subcommand: alias_start())
+        (subcommand: alias_stop())
+        (subcommand: alias_term())
         (after_help: "\nALIASES:\
             \n    apply      Alias for: 'config apply'\
             \n    install    Alias for: 'pkg install'\
+            \n    run        Alias for: 'sup run'\
             \n    setup      Alias for: 'cli setup'\
             \n    start      Alias for: 'sup start'\
```

Comment:
While you're in here, could you change this to be: `start      Alias for: 'svc start'`?